### ISSUE TITLE
bug(<cloudformation>): false positive on API Gateway V2 Stage Access Logging Settings Not Defined

### ISSUE THREAD

**Issue by funkdoodle** [Checkmarx/kics#7466]
kics query 80d45af4-4920-4236-a56e-b7ef419d1941 is incorrectly looking for `MethodSettings.LoggingLevel`.

AWS documentation https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigateway-stage.html#cfn-apigateway-stage-methodsettings states that `MethodSettings` is an array (not a mapping)

The kics example for negative test at https://docs.kics.io/latest/queries/cloudformation-queries/aws/80d45af4-4920-4236-a56e-b7ef419d1941/ is invalid CloudFormation that will not pass cfn-lint.

### Expected Behavior

The following template does not trigger 80d45af4-4920-4236-a56e-b7ef419d1941 
```
Resources:
  Prod:
    Type: AWS::ApiGateway::Stage
    Properties:
      StageName: Prod
      Description: Prod Stage
      RestApiId: !Ref MyRestApi
      DeploymentId: !Ref TestDeployment
      DocumentationVersion: ""
      MethodSettings:
        - LoggingLevel: "INFO"
      AccessLogSetting:
        DestinationArn: "dest"
        Format: "format"
```

### Actual Behavior

Query 80d45af4-4920-4236-a56e-b7ef419d1941 is currently triggering even when access logging is configured.

### Steps to Reproduce the Problem

See above

### Specifications

N/A
